### ISSUE TITLE
[Feature]: Search drawer network and highlight correct drawer

### ISSUE THREAD

**Issue by dhouck** [MattiDragon/ExtendedDrawers#96]
### Description of the feature

Add a GUI to Access Points, (or an Index Block or something) that lets you see what items are in the network.  By default it should not let you take any items (like current behavior) but it would be nice if that were a config option.  It should, however, support a search box where you type an item name and it will highlight any drawers that have that item.

### How does it make the mod better?

When you have lots of drawers, it can be hard to remember where you put whatever item you are currently looking for.  It’s *visible* so it’s easier than vanilla chests, but it can still require a lot of looking.  This would make that much easier.

For my use case I might be able to 80/20 it with something that lets me withdraw items from the access point, but I don’t want to bring in all of AE2 and don’t know other ME-compatible mods which do less, so for me in particular a mod suggestion would probably be easier.  But even so I think the feature as described would be useful; I don’t think anything that uses the Fabric Item Transfer API or the ME interface would be able to identify which drawer something is in.

**Comment by MattiDragon** [comment]
This is out of scope for this mod. I've tried to avoid guis (the only gui is the limiter setup). This would probably be something an addon could feasiably do if someone put the time and effort into it. If you want something lighter than AE2 for digital storage I'd recommend looking into [toms simple storage mod](https://modrinth.com/mod/toms-storage). It offers a gui for taking out items, but is more in style with vanilla and extended drawers

**Comment by dhouck** [comment]
Fair enough.  I’ll check that mod out; it looks good at first glance at least.

I looked for a search mod before posting the issue and didn’t see anything good, but I tried again afterwards and found [Where Is It](https://github.com/JackFred2/WhereIsIt/), which seems to do almost exactly what I want.  It uses a radius from the player instead of the drawer network, and does not show a list of what’s in the network, but otherwise works great (especially if you have EMI or something to help search with).
